app/uk/gov/hmrc/securemessage/connectors/EISConnector.scala

```diff
@@ -61,8 +68,11 @@ class EISConnector @Inject()(httpClient: HttpClient, servicesConfig: ServicesCon
       )
       .flatMap { response =>
         response.status match {
-          case NO_CONTENT => Future(Right(()))
+          case NO_CONTENT =>
+            auditMessageForwarded(EventTypes.Succeeded, queryResponse, NO_CONTENT)
+            Future(Right(()))
           case code =>
+            auditMessageForwarded(EventTypes.Failed, queryResponse, code)
```

Comment:
Is this a side-effecting method that returns `Unit`?

Reply:
Indeed. I have now made this explicit.